posthog/hogql_queries/insights/lifecycle_query_runner.py

```diff
@@ -95,6 +95,7 @@ def to_query(self) -> ast.SelectQuery | ast.SelectUnionQuery:
     def to_actors_query(
         self, day: Optional[str] = None, status: Optional[str] = None
     ) -> ast.SelectQuery | ast.SelectUnionQuery:
+        # should I update this?
```

Comment:
I could see this being a followup PR, however yes, somebody one day should. This is the query that opens when you click on one of the bars on the chart. I don't think it's that many extra steps to get this working.